static/styles/settings.css

```diff
@@ -276,6 +276,14 @@ td .button {
         z-index: 1;
     }
 
+    #id_realm_authentication_methods {
+        width: 20%;
+
+        td {
+            border: none;
+        }
+    }
```

Comment:
Thinking about this more, I think we should switch this component from using table HTML to just being a series of `settings_checkbox` components.  Can you attempt that as a follow-up pull request?

Reply:
sure !, I'll raise a PR for this asap.